README.md

```diff
@@ -83,19 +83,25 @@ See also Troubleshooting.
 
 -  Run this:
 
-      `github_changelog_generator -u github_username -p github_project`
-      `github_changelog_generator  github_username/github_project`
+	   $ github_changelog_generator -u github_username -p github_project
+	   
+	or
```

Comment:
Perhaps we can make a note here, about versions.

```
    or, on the 1.14.x (current stable release)
```

Reply:
this was what brought you all the way here, to fixing the documentation.

(Thanks for that!)

Reply:
Or just remove the old syntax? When people download the latest version, they dont need to know old stuff.

Reply:
The README describes things which are yet to be released.

The visitor may have installed the 1.14.x (stable), and not 1.15.x-pre. (pre-release).

Using `gem install ...` will install stable.

Reply:
So, they _may_ need to know this difference.

Reply:
Changed it.